app/scheduler_bootstrap.sh

```diff
@@ -1,5 +1,4 @@
-yum install -y wget
-wget https://bootstrap.pypa.io/get-pip.py
+curl -O https://bootstrap.pypa.io/get-pip.py
 python get-pip.py
 
 pip install azure-storage
```

Comment:
I don't think we can use pip in BAC Linux VMs - we should just make sure we call out the dependency on azure-storage when we work with them.